graphql_compiler/schema_transformation/utils.py

```diff
@@ -263,3 +284,84 @@ def check_ast_schema_is_valid(ast):
 
     query_type = get_query_type_name(schema)
     visit(ast, CheckQueryTypeFieldsNameMatchVisitor(query_type))
+
+
+def is_property_field(selection):
```

Comment:
Maybe `is_property_field_ast` to communicate to the reader of the code that this function deals with ASTs and not field names? I believe there is a similarly-named function to this that does this for field names.